src/components/TodoCard/TodoCard.jsx

```diff
@@ -1,12 +1,14 @@
 import React from 'react';
+import DeleteButton from '../DeleteButton/DeleteButton';
 import './TodoCard.css'
 
-function TodoCard({ todo }) {
+function TodoCard({ todo, onDelete }) {
 
     return (
         <div key={todo?.id} className='todo-card'>
             <p className='todo-card--number'>{todo?.id}</p>
             <p className='todo-card--text'>{todo?.text}</p>
+            <DeleteButton id={todo.id} onDelete={onDelete} />
```

Comment:
you don't need to pass id as extra props.
You can pass it as parameter to `onDelete` instead.

```
<DeleteButton onDelete={() => onDelete(id)} />
```